README.md

```diff
@@ -116,3 +116,23 @@ You can use a couple of different formats, with restrictions:
 2. `A:B` --> `ETH:USDT`
 3. `A-B` --> `ETH-USDT`
 4. `AB` --> `ETHUSDT`
+
+
+## Development
+
+Regenerate protobuf files (uses a Docker builder for consistency):
+
+```
+$ make proto
+```
+
+Unit tests:
+
+```
+$ make unit
+```
+
+Integration tests:
+```
+$ make test
```

Comment:
FYI: As integration was replaced with examples I added 3 make scripts in my PR :
```
.PHONY: grpc-examples
grpc-examples:
	 go run ./examples/grpcclient/main.go

.PHONY: http-examples
http-examples:
	 go run ./examples/httpclient/main.go

.PHONY: ws-examples
ws-examples:
	 go run ./examples/wsclient/main.go
```

Reply:
:+1: can you update the README in your PR?